app/views/devise/registrations/new.html.erb

```diff
@@ -1,34 +1,36 @@
-<h2>Sign up</h2>
-
+<h2 class="log-in-header">Sign up</h2>
+<div class="login-form">
 <%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
   <%= render "devise/shared/error_messages", resource: resource %>
 
     <div class="field">
     <%= f.label :name %><br />
     <%= f.text_field :name %>
   </div>
-
+ <br>
   <div class="field">
     <%= f.label :email %><br />
     <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
   </div>
-
+  <br>
   <div class="field">
     <%= f.label :password %>
     <% if @minimum_password_length %>
     <em>(<%= @minimum_password_length %> characters minimum)</em>
     <% end %><br />
     <%= f.password_field :password, autocomplete: "new-password" %>
   </div>
-
+  <br>
   <div class="field">
     <%= f.label :password_confirmation %><br />
     <%= f.password_field :password_confirmation, autocomplete: "new-password" %>
   </div>
-
+  <br>
   <div class="actions">
-    <%= f.submit "Sign up" %>
+    <%= f.submit "Sign up", class:"classic-btn" %>
   </div>
 <% end %>
-
+<br>
 <%= render "devise/shared/links" %>
+
+</div>
```

Comment:
- [ ] I get this error page when I sign up as a user. Kindly rectify it. The user is actually created but it shows this unnecessary error.

  ![nfjf](https://user-images.githubusercontent.com/54848363/201176683-2cc03517-4397-4b23-8210-77a59ea3869d.jpg)